modules/cluster-entitlements.adoc

```diff
@@ -39,7 +39,7 @@ ifdef::openshift-origin,openshift-enterprise[]
 
 [IMPORTANT]
 ====
-If your cluster cannot have direct internet access, you can perform a restricted network installation on infrastructure that you provision. During that process, you download the content that is required and use it to populate a mirror registry with the packages that you need to install a cluster and generate the installation program. With some installation types, the environment that you install your cluster in will not require internet access. Before you update the cluster, you update the content of the mirror registry.
+If your cluster cannot have direct internet access, you can perform a restricted network installation on some types of infrastructure that you provision. During that process, you download the content that is required and use it to populate a mirror registry with the packages that you need to install a cluster and generate the installation program. With some installation types, the environment that you install your cluster in will not require internet access. Before you update the cluster, you update the content of the mirror registry.
```

Comment:
Should we use _If your cluster does note have..._ ? _cluster cannot have_ doesn't sound right.

Also, _With some installation types, the environment that you install your cluster in will not require internet access_ sounds redundant here. We've established that the cluster doesn't have internet access in the first sentence.

Reply:
With some installation types, you have to have some internet access to your cluster for the APIs to work.